README.md

```diff
@@ -2,16 +2,52 @@
 
 This repo is used to setup an Auth service for the [data-store](https://github.com/databiosphere/data-store).
 
-
 Setup Steps
 
 1. configure a tenant with Auth0
 1. set the secret with the tenant information in the aws secret-store [Secret Setup](#Secret-Setup)
 1. deploy infra -> sets up all the stuff. more here later. 
 
 
+### Terraform Installation
+
+In order to deploy the infra in this repository, you will need to install Terraform and an Auth0 Terraform
+provider. We use this Auth0 terraform provider: [alexkappa/terraform-provider-auth0](https://github.com/alexkappa/terraform-provider-auth0).
+
+To install terraform, download the Linux binary, unzip it to a directory on your path, and test it is available:
```

Comment:
can osx users use the linux binary? i downloaded the Darwin ones.... 🤔

Reply:
Yeah, I was gonna call out Linux and Mac separately but I figured `s/linux/darwin/` was easy enough that I could leave it out. I'll add a note.

Reply:
I added a nice little trick

```
export OS=$(uname -s | tr '[:upper:]' '[:lower:]')
```

to get `linux` on Linux and `darwin` on Mac.

Reply:
Resolved with 75848e4